examples/ipython/demo.md

```diff
@@ -45,29 +59,30 @@ the Jupyter notebook for local interaction as follows.
     nix-shell -I nixpkgs="https://github.com/NixOS/nixpkgs-channels/archive/cc6cf0a96a627e678ffc996a8f9d1416200d6c81.tar.gz" \
     -p "python3.withPackages(ps: [ps.notebook ps.numpy ps.matplotlib ps.pandas ps.pyarrow ps.networkx])"
     ```
-- With Homebrew
-    ```sh
-    brew install jupyter numpy matplotlib pandas apache-arrow networkx
-    ```
-- Via Virtual Env
+- With `venv`
     ```sh
-    virtualenv --system-site-packages venv
+    python3 -m venv venv
     source venv/bin/activate
-    python3 -m pip install -r requirements.txt
+    # Verify that the python from venv is used:
+    which python
+    python -m pip install -r requirements.txt
+    ipython kernel install --user --name=venv
     ```
 
-Next, install `pyvast` with the following command. That can also be done in a
-virtual env.
+As long as the demo is started from this directory, the step of installing
+`pyvast` can be skipped, because the working tree copy is available locally.
+If you want to install `pyvast` nonethelesss, run the following command:
 
 ```sh
 cd pyvast
-python3 setup.py install
+python setup.py install
```

Comment:
When invoking
```
python setup.py install
```

I am getting this error:

```
/usr/local/Cellar/python/3.7.6_1/Frameworks/Python.framework/Versions/3.7/Resources/Python.app/Contents/MacOS/Python: can't open file 'setup.py': [Errno 2] No such file or directory
```

Reply:
I just verified this on macOS, @0ortmann do you have an idea?

Reply:
I verified that all the above installation steps work on my machine.

@mavam the error message you posted refers to a `file not found` error. Did you change directory to the `pyvast` folder before executing?

Reply:
The symlink changed from `../../pyvast` to `../../pyvast/pyvast` but the command needs to be executed in the former directory.